InnerEye/ML/visualizers/plot_cross_validation.py

```diff
@@ -182,6 +177,13 @@ def get_short_name(self, run_or_id: Union[Run, str]) -> str:
                 self.short_names[run_or_id] = run_id
         return self.short_names[run_id]
 
+    @property
```

Comment:
nitpicking: should be plural, execution_modes
nitpicking: I would not use a property here. Properties are, in my view, meant to read out simple data from an object.

Reply:
updated